tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/GitStore.cs

```diff
@@ -122,12 +123,14 @@ public class GitStore : IAssetsStore
         /// Resets a cloned assets repository to the default contained within the assets.json targeted commit.
         /// </summary>
         /// <param name="pathToAssetsJson"></param>
+        /// <param name="consoleWrapper"></param>
         /// <returns></returns>
         // This should only ever be called by the user?
-        public async Task Reset(string pathToAssetsJson) {
+        public async Task Reset(string pathToAssetsJson, IConsoleWrapper consoleWrapper) 
```

Comment:
Please add `IConsoleWrapper consoleWrapper` to the _constructor_ of the GitStore.

I don't think it should be passed to every `Reset` command.

Reply:
Fixed with next commit.